distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc

```diff
@@ -143,10 +143,10 @@ __Coordinates must be in *lon-lat* coordinate order__
 |
 
 |datatype
-|The generic type(s) of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#terms-type[Dublin Core terms-type] and extended to include other DDF supported types.
+|The generic type(s) of the resource including the http://dublincore.org/documents/dcmi-type-vocabulary/[Dublin Core terms-type]. DCMI Type term labels are expected here, as opposed to term names.
```

Comment:
❓ Do you really need this comma here: "...expected here, as opposed to term..."?  I think you can remove it.

Reply:
Fixed